vips/vip-xxx/bsctestnet.ts

```diff
@@ -0,0 +1,127 @@
+export const MARKET_CAPS_RISK_STEWARD = "0x05968239d978601146D3e50eE1F29e8571249fB0";
+
+
+export const IL_COMPTROLLERS = [
+  LIQUID_STAKING_BNB_COMPTROLLER,
+  DEFI_COMPTROLLER,
+  STABLECOIN_COMPTROLLER,
+  GAMEFI_COMPTROLLER,
+  BTC_COMPTROLLER,
+  TRON_COMPTROLLER,
+  LIQUID_STAKING_ETH_COMPTROLLER,
+  MEME_COMPTROLLER
+]
+const TEN_MINUTES = 60 * 10;
+export const vipXXX = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-XXX Configure Risk Steward Access Control",
+    description: `If passed, this VIP will permission governance to set configurations on the RiskStewardReceiver, the RiskStewardReceiver to forward validated updates the the MarketCapsRiskSteward, and the MarketCapsRiskSteward to update market caps on comptrollers.
+This proposal will also set the risk parameter configs for updating the supplyCap and borrowCap configurations on the RiskStewardReceiver.
+
+The following permissions will be granted:
+- Governance can call setRiskParameterConfig on the RiskStewardReceiver
+- Governance can call toggleConfigActive on the RiskStewardReceiver
+- Governance can call pause on the RiskStewardReceiver
+- Governance can call unpause on the RiskStewardReceiver
+- RiskStewardReceiver can call processUpdate on the MarketCapsRiskSteward
+- Governance can call setMaxIncreaseBps on the MarketCapsRiskSteward
+- MarketCapsRiskSteward can call _setMarketSupplyCaps on the BNB_CORE_COMPTROLLER
+- MarketCapsRiskSteward can call _setMarketBorrowCaps on the BNB_CORE_COMPTROLLER
+- MarketCapsRiskSteward can call setMarketSupplyCaps on IL comptrollers
+- MarketCapsRiskSteward can call setMarketBorrowCaps on IL comptrollers
+
+Risk Paramater Configs:
+- SupplyCap: RiskSteward: ${MARKET_CAPS_RISK_STEWARD}, Debounce: ${TEN_MINUTES}, Active: true
+- BorrowCap: RiskSteward: ${MARKET_CAPS_RISK_STEWARD}, Debounce: ${TEN_MINUTES}, Active: true
+
+VIP simulation: [https://github.com/VenusProtocol/vips/pull/459](https://github.com/VenusProtocol/vips/pull/XXX)`,
+    forDescription: "Execute this proposal",
+    againstDescription: "Do not execute this proposal",
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER, "setRiskParameterConfig(string,address,uint256)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER, "toggleConfigActive(string)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER, "pause()", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER, "unpause()", NORMAL_TIMELOCK],
+      },
```

Comment:
I think Fast-track and Critical timelocks should also be authorized to call these functions